### ISSUE TITLE
Sanity check on base URL

### ISSUE THREAD

**Issue by Rafiot** [MISP/MISP#3273]
If the admin only sets an IP in the field, everything else goes terribly wrong after that. It might be good to make sure the entry starts with `http://`, or prepend it automatically.

Potentially, it would be better to do a connexion check, but I'm not sure it is feasible or not.

**Comment by iglocska** [comment]
Hmm good point.

**Comment by iglocska** [comment]
Ah ok, just checked, it's enforced the UI but not the API. Can do something about it.

**Comment by Rafiot** [comment]
Last week, someone used the web interface, put the IP, and then, everything was wrong ;) 

So that's weird, is it a very recent change?

**Comment by iglocska** [comment]
hmph. Via the interface it should work o.O Will have a look.

**Comment by iglocska** [comment]
ah indeed, it's correct. It should work but it should flag it as a critical issue. Hmph, we don't differentiate between blocking changes and changes that just raise an error.

**Comment by Rafiot** [comment]
Yeah, this one will cause hilarious issues and should be reverted

**Comment by iglocska** [comment]
Then we need to differentiate between different level of issues then. For example, an empty baseurl also raises an issue but shouldn't be blocking.

**Comment by Rafiot** [comment]
True.

**Comment by SteveClement** [comment]
What does: critical_error##COMMA##block
generally mean?
